resources/web/map/js/trajectory.js

```diff
@@ -102,22 +132,20 @@ new QWebChannel(qt.webChannelTransport, (channel) => {
         }
     });
 
-    chn.recvPos.connect((id, lat, lng) => {
-        const pos = [lat, lng];
-
-        data[id].features.push({
-            type: 'Feature',
-            geometry: {
-                type: 'Point',
-                coordinates: pos
-            }
-        });
-        if (!map) return
+    chn.recvPos.connect((id, lat, lng, hAcc, vAcc) => {
+        const pos = [lat, lng],
+            RADIUS_MIN = 0.0002,
```

Comment:
hard cap minimum to be 20cm.
rendering too small is difficult for zooming out.

Reply:
Not sure I understand, if we go to a certain zoom, then the accuracy circles should probably just be removed rather than forcing them to be a particular size.  Or, we can establish a metric (based on mapping meters to pixels) that would dictate whether it would be visible or not.

Reply:
yea... rendering even smaller is extremely difficult to see on the map.

i think drongo has it so the size gets bigger if accuracy decreases. 
** (accuracy circle is the same point, not an additional overlay that gets drawn around the location)

Reply:
@dgburr Any feedback here?

Reply:
agree, shouldn't cap the min size, it is better to just not show it if its too small to render

Reply:
adjusted

Reply:
I can confirm that Drongo does not enforce a minimum cap either: https://github.com/swift-nav/ats/blob/master/drive-exe/swift/map/scene/render.py#L36